### ISSUE TITLE
Support for JSX syntax

### ISSUE THREAD

**Issue by LuoZijun** [ratel-rust/ratel-core#81]
Hi, Have plan to support [JSX](https://reactjs.org/docs/jsx-in-depth.html) syntax ?

**Comment by maciejhirsz** [comment]
Yup! There is no performance cost to non-JSX code in adding it since expressions go through lookup tables on first token :).
